addon/services/text-measurer.js

```diff
@@ -39,5 +39,14 @@ export default Ember.Service.extend({
       }
     }
     return lines;
+  },
+
+  fitTextSize(string, maxWidth, font = null) {
```

Comment:
This method is useful. But I think that is basically the same calculation I'm doing in the 2nd example in the dummy app. Could you refactor that example to use this new method instead of adding a new example?